gulpfile.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const gulp = require('gulp')
+const multiaddr = require('multiaddr')
+const Node = require('./src').Node
```

Comment:
you are using the same libp2p-ipfs-browser to fire the Node.js node (which does nothing on swarm)